docs/release-notes/1.1.0/release-1.1.0.md

```diff
@@ -0,0 +1,116 @@
+  the file path as a string in IDataView and then the image would be loaded
+  using a transform. This resulted in the closure of the following github
+  issues:  [3162](https://github.com/dotnet/machinelearning/issues/3162),
+  [3723](https://github.com/dotnet/machinelearning/issues/3723),
+  [3369](https://github.com/dotnet/machinelearning/issues/3369),
+  [3274](https://github.com/dotnet/machinelearning/issues/3274),
+  [445](https://github.com/dotnet/machinelearning/issues/445),
+  [3460](https://github.com/dotnet/machinelearning/issues/3460),
+  [2121](https://github.com/dotnet/machinelearning/issues/2121),
+  [2495](https://github.com/dotnet/machinelearning/issues/2495),
+  [3784](https://github.com/dotnet/machinelearning/issues/3784) and was a much
+  requested feature by the users.  
+
+    [Sample 1](https://github.com/dotnet/machinelearning/blob/02a857a7646188fec2d1cba5e187a6c9d0838e23/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToGrayScaleInMemory.cs) | [Sample 2](https://github.com/dotnet/machinelearning/blob/02a857a7646188fec2d1cba5e187a6c9d0838e23/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingWithInMemoryCustomType.cs)
+
+- **Super-Resolution based Anomaly Detector**  
+   [PR#3693](https://github.com/dotnet/machinelearning/pull/3693) adds a new
+   anomaly detection algorithm to the time series nuget. This algorithm is based
+   on Super-Resolution using Deep Convolutional Networks and also got accepted
+   in KDD'2019 conference as oral presentation. One of the advantages of this
+   algorithm is that it does not require any prior training and based on
+   benchmarks using grid parameter search to find upper bounds it out performs
+   the IID and SSA based anomaly detection algorithms in accuracy. This
+   contribution comes from the [Azure Anomaly
+   Detector](https://azure.microsoft.com/en-us/services/cognitive-services/anomaly-detector/)
+   team.
+
+    Algo | Precision | Recall | F1 | #TruePositive | #Positives | #Anomalies | Fine tuned   parameters
+    -- | -- | -- | -- | -- | -- | -- | --
+    SSA (requires training) | 0.582 | 0.585 | 0.583 | 2290 | 3936 | 3915 | Confidence=99,   PValueHistoryLength=32, Season=11, and use half the data of each series to do   the training.
+    IID | 0.668 | 0.491 | 0.566 | 1924 | 2579 | 3915 | Confidence=99,   PValueHistoryLength=56
+    SR | 0.601 | 0.670 | 0.634 | 2625 | 4370 | 3915 | WindowSize=64,   BackAddWindowSize=5, LookaheadWindowSize=5, AveragingWindowSize=3,   JudgementWindowSize=64, Threshold=0.45
+
+    [Sample 1](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectAnomalyBySrCnn.cs) | [Sample 2](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectAnomalyBySrCnnBatchPrediction.cs)
+
+- **Time Series Forecasting**  
```

Comment:
Afaik, this feature is being released in Preview? If so, specify, otherwise looks like it is being released as 1.1 GA.

Reply:
Same as above